airbyte-integrations/connectors/source-s3/integration_tests/spec.json

```diff
@@ -177,6 +177,11 @@
             "default": "",
             "type": "string"
           },
+          "use_aws_default_credential_provider_chain": {
```

Comment:
Could you please create an `authentication_method` oneOf field which for the user to chose between *Default credential provider* and *Access Key/Secret Key*?

Reply:
@alafanechere I am a little stuck here on how to implement this without causing a breaking change. 

We have Default Credential Provider, Access Key/Secret Key and no authentication. The current behaviour defaults to Access Key/Secret Key if those values are included and falls back to no authentication if not included. 
When introducing oneOf field `authentication_method`, I'm not sure what the default value can be set to.

Reply:
What do you think of removing `use_aws_default_credential_provider_chain` field and changing the description of Access Key/Secret Key to explain that if not set it will fall back to the default boto credential chain?

Reply:
Since we need to support those who don't use any credentials (e.g. those who use it with non-AWS storage or those who don't need any auth since they are accessing public buckets), I don't think this will work. In this case, the default boto credential chain won't be able to locate any credentials.